packages/legend-query-builder/src/stores/QueryBuilderState.ts

```diff
@@ -371,6 +374,20 @@ export abstract class QueryBuilderState implements CommandRegistrar {
     this.setRuntimeValue(val);
   }
 
+  getParameterExecutionValues(): Map<string, ValueSpecification> | undefined {
```

Comment:
First time I read this, I find it confusing since what does it really contrast to? let's simplify the name to `getCurrentParameterValues`